src/util/externalCommand.ts

```diff
@@ -80,6 +84,8 @@ class ExternalCommand {
   public async run(command: string, stdin: string = ''): Promise<string> {
     command = this.expandCommand(command);
     this.previousExternalCommand = command;
+    // combines stdout and stderr (compatible for all platforms)
+    command += ' 2>&1';
```

Comment:
You removed this in your previous PR - is it necessary?

Reply:
Yeah it's necessary, I previously removed it because I thought the child process pipes would maintain the correct ordering of stdout/stderr, but they don't - so it's best to redirect both to stdout to get the right order.